### DISCUSSION TITLE
change custom walking animation when triggered

### DISCUSSION THREAD

**Discussion by Sinanaltundag** [Annoraaq/grid-engine#332] [Q&A]
I use isometric map example so custom walking animation. Load both spritesheet on preload;
```
        this.load.spritesheet("player", "assets/iso_char.png", { frameWidth: 15, frameHeight: 32, });
        this.load.spritesheet("player2", "assets/iso_char_light.png", {frameWidth: 15, frameHeight: 32, });
```
then when triggered on event:

```
playerSprite.setTexture("player2", 0);
createPlayerAnimation.call( this, "up-left", 16, 19, "player2" ); // I configure function for new sprite sheet
```
setTexture change character but walking animation not changed

I want to change 2 different characters with same size, Can any one help, how can I do?

**Comment by Annoraaq** [comment]
For future readers of this question: I made a proposal in https://github.com/Annoraaq/grid-engine/issues/333#issuecomment-1326754229.
